packages/components/tests/dummy/app/templates/components/empty-state.hbs

```diff
@@ -0,0 +1,88 @@
+  <p class="dummy-paragraph">The EmptyState component is a block-yielding component that contains three child
+    components: a header, body and footer. Each child component is also a block yield. While none of the child
+    components are explicitly required, use of at least the header or body components are recommended.</p>
+  <h4 class="dummy-h4">Basic use</h4>
+  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code="
+      <Hds::EmptyState as |E|>
+        <E.Header>Meaningful Header Text</E.Header>
+        <E.Body>Body text content that provides a meaningful message to the end user.</E.Body>
+        <E.Footer>Footer Content - text, or a link or maybe even a (secondary) button.</E.Footer>
+      </Hds::EmptyState>
+    "
+  />
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::EmptyState as |E|>
+    <E.Header>Meaningful Header Text</E.Header>
+    <E.Body>Body text content that provides a meaningful message to the end user</E.Body>
+    <E.Footer>Footer Content - text, or a link or maybe even a (secondary) button.</E.Footer>
+  </Hds::EmptyState>
+
+  <h4 class="dummy-h4">Header and Body</h4>
+  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code="
+      <Hds::EmptyState as |E|>
+        <E.Header>Header Text</E.Header>
+        <E.Body>Body Text</E.Body>
+      </Hds::EmptyState>
+    "
+  />
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::EmptyState as |E|>
+    <E.Header>Header Text</E.Header>
+    <E.Body>Body Text</E.Body>
+  </Hds::EmptyState>
+</section>
+
+<section>
+  <div class="dummy-design-guidelines">
+    <h3 class="dummy-h3" id="design-guidelines"><a href="#design-guidelines" class="dummy-link-section">§</a>
+      Design guidelines</h3>
+    <p class="dummy-paragraph">To be released at a future date.</p>
+  </div>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
+  <p class="dummy-paragraph">This component and its child components are rendered as blocks for developers to place
+    content into. Developers should ensure that the content is conformant with WCAG 2.1 AA Success Criteria.</p>
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
+  <h4 class="dummy-h4">Basic use:</h4>
+  <Hds::EmptyState as |E|>
+    <E.Header>Meaningful Header Text</E.Header>
+    <E.Body>Body text content that provides a meaningful message to the end user</E.Body>
+    <E.Footer>Footer Content - text, or a link or maybe even a (secondary) button.</E.Footer>
+  </Hds::EmptyState>
```

Comment:
Suggestion/Nit: Would it be helpful to provide one more real-world example from one of the products for the content?